cloudos_cli/__main__.py

```diff
@@ -3415,22 +3415,26 @@ def rm_item(ctx, target_path, apikey, cloudos_url,
 
 
 @datasets.command(name="link")
-@click.argument("s3_path", required=True)
+@click.argument("path", required=True)
 @click.option('-k', '--apikey', help='Your CloudOS API key', required=True)
 @click.option('-c', '--cloudos-url',
               help=(f'The CloudOS url you are trying to access to. Default={CLOUDOS_URL}.'),
               default=CLOUDOS_URL)
+@click.option('--project-name',
+              help='The name of a CloudOS project.',
+              required=False)
 @click.option('--workspace-id', help='The specific CloudOS workspace id.', required=True)
 @click.option('--session-id', help='The specific CloudOS interactive session id.', required=True)
 @click.option('--disable-ssl-verification', is_flag=True, help='Disable SSL certificate verification.')
 @click.option('--ssl-cert', help='Path to your SSL certificate file.')
 @click.option('--profile', help='Profile to use from the config file', default='default')
 @click.pass_context
-def link(ctx, s3_path, apikey, cloudos_url, workspace_id, session_id, disable_ssl_verification, ssl_cert, profile):
+def link(ctx, path, apikey, cloudos_url, project_name, workspace_id, session_id, disable_ssl_verification, ssl_cert, profile):
     """
-    Link a S3 folder to an active interactive analysis.
+    Link a folder (S3 or File Explorer) to an active interactive analysis.
 
-    S3_PATH [path]: the full path to the S3 folder to link. E.g.: 's3://bucket-name/folder/subfolder'\n
+    PATH [path]: the full path to the S3 folder to link or relative to File Explorer. 
+    E.g.: 's3://bucket-name/folder/subfolder', 'Data/Downloads' or 'Data'.
     """
     print(link.__doc__ + '\n')
```

Comment:
```suggestion
``` 
Could we remove this? Having the docstring everytime the link command runs is not consistent with the rest of the datasets command